src/components/toolbox/checkBox/checkBox.css

```diff
@@ -1,10 +1,12 @@
-@import '../app/variablesV2.css';
+@import '../../app/variablesV2.css';
 
 .checkbox {
   display: inline-block;
   width: 16px;
   height: 16px;
   position: relative;
+  z-index: 2;
```

Comment:
Maybe would be good to create some variables for the z-index, so it's easier, something similar to:
```css
--background-index: 0;
--base-index: 3;
--foreground-index: 6;
--overlay-index: 10;
```
So it would be easier to visualize which element is in front and which are in the back.

Reply:
Somehow tried to do this :-)